src/components/AllServicesDropdown/AllServicesDropdown.scss

```diff
@@ -1,27 +1,31 @@
-@import '~@patternfly/patternfly/sass-utilities/_all';
+@import '~@patternfly/patternfly/sass-utilities/scss-variables';
 
 .chr-c-link-service-toggle {
+
   // override menu toggle extra bottom border
-  &.pf-v5-c-menu-toggle::after {
+  &.pf-v6-c-menu-toggle::after {
     border-bottom: none;
   }
-  &.pf-v5-c-menu-toggle:hover::after, &.pf-m-expanded::after {
-    border-bottom: var(--pf-v5-global--BorderWidth--lg) solid var(--pf-v5-global--info-color--100);
+
+  &.pf-v6-c-menu-toggle:hover::after,
+  &.pf-m-expanded::after {
+    border-bottom: var(--pf-t--global--border--width--extra-strong) solid var(--pf-t--global--border--color--status--info--default);
   }
 }
 
 .chr-c-page__services-nav-dropdown-menu {
-  @media screen and (min-width: $pf-v5-global--breakpoint--2xl) {
-    top: 70px;
-    height: 100vh;
```

Comment:
We should take note of this as there were some specific reasons why this property exists. Or is this something that is replaced internal by the page layout in PF now?

Reply:
I believe that this was removed based on feedback from design. I want to say an issue with nested scrollable areas & double scroll bars on some of the services pages.

What was the reason the 100vh height was needed originally? I might be able to accomplish that original goal and not introduce the nested scrolling issue with a slightly different approach.

Reply:
I don't remember TBH. But I want to keep this in mind. In case we get some reports.